crates/ironrdp-web/src/clipboard/mod.rs

```diff
@@ -24,13 +25,13 @@ use ironrdp::cliprdr::pdu::{
 use ironrdp_cliprdr_format::bitmap::{dib_to_png, dibv5_to_png, png_to_cf_dibv5};
 use ironrdp_cliprdr_format::html::{cf_html_to_plain_html, plain_html_to_cf_html};
 use ironrdp_core::{impl_as_any, IntoOwned};
-use transaction::{ClipboardContent, ClipboardContentValue};
+use transaction::ClipboardContentValue;
 use wasm_bindgen::prelude::*;
 
 use crate::session::RdpInputEvent;
 
 #[rustfmt::skip]
-pub(crate) use transaction::ClipboardTransaction;
+pub(crate) use transaction::{RdpClipboardTransaction, RdpClipboardContent};
```

Comment:
nitpick: General remark/suggestion. The `Rdp` prefix was added to a few types. But it’s a bit inconsistent with the rest.

suggestion: You can keep the original names like this:
```rust
struct ClipboardTransaction;

impl iron_remote_desktop::ClipboardTransaction for ClipboardTransaction {
    // …
}
```

Reply:
It would prefer to have the `Rdp` prefix, but renamed to original names